src/Writers/LaTeXWriter.jl

```diff
@@ -506,6 +506,14 @@ function files!(out::Vector, v::Vector, depth)
     return out
 end
 
+# Tuples comde from `hide(page)` with either
+# (visible, nothing,    page,         children) or
+# (visible, page.first, pages.second, children)
+function files!(out::Vector, v::Tuple, depth)
+    files!(out, v[2] == nothing ? v[3] : v[2] => v[3], depth)
```

Comment:
just a suggestion: for v0.7-only code,
```julia
function files!(out::Vector, 
                (_, firstpage, secondpage, children)::NamedTuple{(:visible, :firstpage, :secondpage, :children)},
                depth)
    files!(out, firstpage == nothing ? secondpage : firstpage => secondpage, depth)
    files!(out, children, depth)
end
```
could be more informative without requiring an explicit type (with the same change in the caller).